src/command.zig

```diff
@@ -16,32 +18,100 @@ pub const Command = enum {
     Unknown,
 };
 
-pub fn handle_commands(cmd: Command, params: ?[]const u8) !void {
-    switch (cmd) {
-        .List => {
-            try handle_list();
-        },
-        .Install => {
-            try install_version(params);
-        },
-        .Use => {
-            try use_version(params);
-        },
-        .Default => {
-            try set_default();
-        },
-        .Version => {
-            try get_version();
-        },
-        .Help => {
-            try display_help();
-        },
-        .Unknown => {
-            try handle_unknown();
-        },
+const CommandData = struct {
+    cmd: Command = .Unknown,
+    param: ?[]const u8 = null,
+};
+
+const CommandOption = struct {
+    short_handle: ?[]const u8,
+    handle: []const u8,
+    cmd: Command,
+};
+
+/// now all available commands
+const command_opts = [_]CommandOption{
+    .{ .short_handle = "ls", .handle = "list", .cmd = Command.List },
+    .{ .short_handle = "i", .handle = "install", .cmd = Command.Install },
+    .{ .short_handle = null, .handle = "use", .cmd = Command.Use },
+    .{ .short_handle = null, .handle = "--version", .cmd = Command.Version },
+    .{ .short_handle = null, .handle = "--help", .cmd = Command.Help },
+    .{ .short_handle = null, .handle = "--default", .cmd = Command.Default },
+};
+
+/// parse command and handle commands
+pub fn handleCommand(params: []const []const u8) !void {
```

Comment:
```suggestion
pub fn handle_command(params: []const []const u8) !void {
```

I did a previous PR, where i tried to follow tigerstyle. https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md#naming-things

I like the approach of keeping it simple and also use snake_case for fn.